sass/layout/_sidebar-left.scss

```diff
@@ -963,11 +963,7 @@ $sidebarOpacityAnimationDuration: 0.15s;
     /* Content */
     .SidebarChannelGroup_content {
         min-height: 2px;
-        margin-bottom: 14px;
-
-        &.hasFollowingSibling {
```

Comment:
Could you please remove this https://github.com/mattermost/mattermost-webapp/blob/022ced2c95214fc225b24e2f9b0ceccbdd42f7e4/components/sidebar/sidebar_category/sidebar_category.tsx#L411

Reply:
Addressed